src/ui_basic/window.cc

```diff
@@ -221,7 +228,12 @@ void Window::layout() {
 		center_panel_->set_pos(Vector2i::zero());
 		center_panel_->set_size(get_inner_w(), get_inner_h());
 	}
-	button_close_->set_pos(Vector2i(get_w() - 2 * TP_B_PIXMAP_THICKNESS, -TP_B_PIXMAP_THICKNESS));
+	button_close_->set_pos(Vector2i(
+	   get_w() + kWindowTitlebarButtonsYPos - TP_B_PIXMAP_THICKNESS, kWindowTitlebarButtonsYPos));
+	button_pin_->set_pos(Vector2i(kWindowTitlebarButtonsYPos, kWindowTitlebarButtonsYPos));
+	button_minimize_->set_pos(Vector2i(
+	   kWindowTitlebarButtonsYPos + kWindowTitlebarButtonsSize + kWindowTitlebarButtonsSpacing,
```

Comment:
How about
```suggestion
	   button_pin_->get_x() + button_pin_->get_w() + kWindowTitlebarButtonsSpacing,
```
I think it makes it more obvious what it means.

Also update the method's comment above.